src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -967,6 +969,7 @@ def test_linux_webapp(self, resource_group):
 
 
 class LinuxWebappSSHScenarioTest(ScenarioTest):
+    @unittest.skip("Flaky test")
```

Comment:
Why do we skip this test?

Reply:
This test is flaky and should possibly be removed or rewritten at some point. When I reran the test in live mode, it failed in both this branch and on dev